src/main/java/org/junit/experimental/results/ResultMatchers.java

```diff
@@ -34,14 +36,21 @@ public ResultMatchers() {
      * Matches if there are {@code count} failures
      */
     public static Matcher<PrintableResult> failureCountIs(final int count) {
+        return failureCount(equalTo(count));
+    }
+
+    /**
+     * Matches if the number of failures matches {@code countMatcher}
+     */
+    public static Matcher<PrintableResult> failureCount(final Matcher<Integer> countMatcher) {
         return new TypeSafeMatcher<PrintableResult>() {
             public void describeTo(Description description) {
-                description.appendText("has " + count + " failures");
+                description.appendText("has a number of failures matching " + countMatcher);
```

Comment:
I personally don't see anything wrong with the existing description. In theory, changing the description could break some users of `ResultMatchers.failureCountIs()`. Is there some motivation for the change?